plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/TestDefaultJdbcQueryBuilder.java

```diff
@@ -483,16 +488,49 @@ public void testBuildSqlWithTimestamp()
         }
     }
 
+    @Test
+    public void testBuildJoinSql()
```

Comment:
Why the additional test? (shouldn't the functionality of the defaultquerybuilder be unchanged?)

Reply:
There was no coverage for joins so I've added one